lib/message_bus/backends/redis.rb

```diff
@@ -346,7 +346,29 @@ def global_subscribe(last_id = nil, &blk)
       private
 
       def new_redis_connection
-        ::Redis.new(@redis_config.dup)
+        config = @redis_config.filter do |k, v|
+          # This is not ideal, required for Redis gem version 5
+          # redis-client no longer accepts arbitrary params
+          # anything unknown will error out.
+          # https://github.com/redis-rb/redis-client/blob/master/lib/redis_client/config.rb#L21-L39
```

Comment:
Minor nit but we almost never want to point to a branch when leaving links to github because the branch can update in the future and the line numbers may not be accurate. You can get the permanent link if you press `y` while viewing the file on github.